src/applications/disability-benefits/686c-674-v2/config/chapters/veteran-information/veteran-address/veteran-address.js

```diff
@@ -1,86 +1,39 @@
-  addressUISchema,
-  updateFormDataAddress,
-} from '../../../address-schema';
+  addressSchema,
+  addressUI,
+} from 'platform/forms-system/src/js/web-component-patterns';
+// import { updateFormDataAddress } from '../../../address-schema';
+import { generateTitle } from '../../../helpers';
 
-const veteranContactInformationSchema = cloneDeep(
-  veteranInformation.properties.veteranAddress,
-);
-
-veteranContactInformationSchema.properties.veteranAddress = buildAddressSchema(
-  true,
-);
-
-// add confirm email field on the frontend only
-veteranContactInformationSchema.properties['view:confirmEmail'] = {
-  type: 'string',
+export const uiSchema = {
+  veteranContactInformation: {
+    'ui:title': generateTitle('Mailing address'),
+    veteranAddress: {
+      ...addressUI({
+        title: '',
+        labels: {
+          militaryCheckbox:
+            'I live on a United States military base outside of the U.S.',
+        },
+      }),
+    },
+  },
 };
 
 export const schema = {
   type: 'object',
   properties: {
-    veteranContactInformation: veteranContactInformationSchema,
-  },
-};
-
-export const uiSchema = {
-  veteranContactInformation: {
-    veteranAddress: addressUISchema(
-      true,
-      'veteranContactInformation.veteranAddress',
-      () => true,
-    ),
-    phoneNumber: {
-      'ui:required': () => true,
-      'ui:title': 'Phone Number',
-      'ui:options': {
-        widgetClassNames: 'usa-input-medium',
-        updateSchema: () => {
-          return {
-            type: 'string',
-            pattern: '^\\d{10}$',
-          };
-        },
-      },
-      'ui:errorMessages': {
-        pattern: 'Enter a 10-digit phone number without dashes or spaces',
-        minLength: 'Enter a 10-digit phone number without dashes or spaces',
-        required: 'Enter a phone number',
-      },
-    },
-    emailAddress: emailUI(),
-    'view:confirmEmail': {
-      'ui:required': formData =>
-        formData.veteranContactInformation.emailAddress !== undefined,
-      'ui:validations': [
-        (errors, fieldData, formData) => {
-          if (
-            formData?.veteranContactInformation?.emailAddress.toLowerCase() !==
-            formData?.veteranContactInformation?.[
-              'view:confirmEmail'
-            ].toLowerCase()
-          ) {
-            errors.addError('Ensure your emails match');
-          }
-        },
-      ],
-      'ui:title': 'Confirm email address',
-      'ui:options': {
-        expandUnder: 'emailAddress',
-        expandUnderCondition: emailAddress => {
-          return emailAddress;
-        },
+    veteranContactInformation: {
+      type: 'object',
+      properties: {
+        veteranAddress: addressSchema(),
       },
     },
   },
 };
 
-export const updateFormData = (oldFormData, formData) =>
-  updateFormDataAddress(oldFormData, formData, [
-    'veteranContactInformation',
-    'veteranAddress',
-  ]);
+// export const updateFormData = (oldFormData, formData) =>
```

Comment:
does this code need to be preserved?

Reply:
I am preserving this until later in the forms development to decide whether this is still the best practice for updating a user account with the captured information.